### PR TITLE
Fix-173966: Separate persistable state from logic

### PR DESCRIPTION
**Pull request by eduardyuzlikeev**

### CODE REVIEW

**Comment by ilya-dubovskiy** [comment]
Build and tests are GOOD

**Comment by ilya-dubovskiy** [comment]
Build and tests are GOOD

File: app/models/akka/Events.scala [hunk 1]
```diff
@@ -8,13 +8,14 @@ import models.Origin
...
-import models.akka.responder.RidRegistry.LookupRecord
+import models.akka.responder.RidRegistry.{LookupRecord, RidRegistryState}
+import models.akka.responder.SidRegistry.SidRegistryState
 import models.akka.responder._
 import models.rpc.DSAMethod.DSAMethod
 import models.rpc.DSAValue.DSAVal
 
-import collection.mutable.{Map => MutableMap}
-import collection.immutable.{Map => ImmutableMap}
+import collection.mutable.{Map ⇒ MutableMap}
```
Comment (dmitriyisaev) [inline app/models/akka/Events.scala]: O no! Hate that symbols. Maybe would be better to keep => ?

**Review by dmitriyisaev — APPROVED** [review]

**Comment by ilya-dubovskiy** [comment]
Build and tests are GOOD

**Comment by ilya-dubovskiy** [comment]
Build and tests are GOOD

**Comment by ilya-dubovskiy** [comment]
Build and tests are GOOD

**Pull request merged**